src/main/scala/ai/privado/languageEngine/java/tagger/sink/JavaAPITagger.scala

```diff
@@ -154,7 +154,7 @@ class JavaAPITagger(cpg: Cpg) extends ForkJoinParallelCpgPass[RuleInfo](cpg) {
     if (apis.nonEmpty && filteredLiteralSourceNode.nonEmpty) {
       val apiFlows = apis.reachableByFlows(filteredLiteralSourceNode).l
       apiFlows.foreach(flow => {
-        val literalCode = flow.elements.head.originalPropertyValue.getOrElse(flow.elements.head.code)
+        val literalCode = flow.elements.head.code
```

Comment:
@dbMundada don't change anything here,  instead change CfgNode to AstNode for the travesal originalTraversalValue which is presemt in privado language module

Reply:
For reference you can refer the PR raised by Rahul

Reply:
Similar commnet for 2 other places below